cmd/poller/plugin/README.md

```diff
@@ -287,18 +290,74 @@ Same as `exclude_equals`, but will use a regular expression and all matching ins
 Rule syntax:
 
 ```yaml
-exclude_contains: LABEL `REGEX`
+exclude_regex: LABEL `REGEX`
 # label name - regular expression
 ```
 
 Example:
 
 ```yaml
-exclude_equals: vol_type `flexgroup_`
+exclude_regex: vol_type `^flex`
+# all instances, which have label "vol_type" which starts with
+# "flex" will not be exported
+```
+
+## include_equals
+
+Include each instance, if the value of `LABEL` is exactly `VALUE`. Include means that metrics for this instance will be exported.
```

Comment:
Include means that metrics for this instance will be exported and instances that do not match will not be exported.